NewsPortal.BusinessLogic/Interfaces/IRoleService.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace NewsPortal.BusinessLogic.Interfaces
+{
+    public interface IRoleService
```

Comment:
Please, implement RoleService with 'throw new NotImplementedException' in all method bodies;